snowflake_queries/snowflake_key_user_password_login.yml

```diff
@@ -0,0 +1,17 @@
+AnalysisType: scheduled_rule
+Filename: snowflake_key_user_password_login.py
+RuleID: Snowflake.KeyUserPasswordLogin
+Description: >
+  Detect when a user that has key-based authentication configured logs in with a password
+DisplayName: Snowflake user with key-based auth logged in with password auth
+Enabled: false
+ScheduledQueries:
+  - Query.Snowflake.KeyUserPasswordLogin
+Severity: Medium
+Tests:
+  -
+    Name: Key user logged in with Password
+    ExpectedResult: true
+    Log:
+      query_text: "testuser RSAKey Password"
```

Comment:
is this the expected output from the query? What shows in the title output from this test?

Reply:
Changed based on our conversation